data-manipulation/hashing/fnv/hash-data-using-fnv.yml

```diff
@@ -33,3 +40,4 @@ rule:
           - or:
             - mnemonic: imul
             - mnemonic: mul
+            - api: _allmul
```

Comment:
should this be a `function-name` feature?

linter fails:   FAIL: doesn't match on referenced example: Fix the rule logic or provide a different example

Reply:
`function-name` is for matching the name of the current function. i think `api` is correct here.